src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -100,7 +100,7 @@ codet java_bytecode_instrumentt::throw_exception(
     generate_class_stub(
       exc_name,
       symbol_table,
-      get_message_handler());
+      message_handler);
```

Comment:
This is not an auto-merge, rather a bug fix (the message handler was never set for `java_bytecode_instrumentt`, and upstream added an assertion to ensure it *was* set on a call to `get_message_handler()`. However I've changed this to set it instead.